README.rst

```diff
@@ -28,6 +28,12 @@ Features
   the fake server, from your webpage
 - **supports webhooks** that you can customize using a special API route
 
+Limitations
+--------
```

Comment:
Please add the correct number of `-` and it will be good to go!

Reply:
Ah, like a subheading of features, no problem

Reply:
```
Limitations
-----------
```

Usually in RST, we write the underline a long as the title line itself.

More info at https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Reply:
Ah cool, no worries